cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometryFeatureDataset.java

```diff
@@ -0,0 +1,240 @@
+    }
+
+    public String getDescription() {
+        String desc = ncd.findAttValueIgnoreCase(null, "description", null);
+        if (desc == null)
+            desc = ncd.findAttValueIgnoreCase(null, CDM.HISTORY, null);
+        return (desc == null) ? getName() : desc;
+    }
+
+    public String getLocation() {
+        return (ncd != null) ?  ncd.getLocation() : "";
+    }
+
+    @Override
+    public CalendarDateRange getCalendarDateRange() {
+        return null;
+    }
+
+    @Override
+    public CalendarDate getCalendarDateStart() {
+        return null;
+    }
+
+    @Override
+    public CalendarDate getCalendarDateEnd() {
+        return null;
+    }
+
+    @Override
+    public LatLonRect getBoundingBox() {
+        return null;
+    }
+
+    public List<Attribute> getGlobalAttributes() {
+        return ncd.getGlobalAttributes();
+    }
+
+    public Attribute findGlobalAttributeIgnoreCase(String name) {
+        return ncd.findGlobalAttributeIgnoreCase(name);
+    }
+
+    @Override
+    public List<VariableSimpleIF> getDataVariables() {
+        return null;
+    }
+
+
+    public VariableSimpleIF getDataVariable(String shortName) {
+        return ncd.getRootGroup().findVariable(shortName);
+    }
+
+    public NetcdfFile getNetcdfFile() {
+        return ncd;
+    }
+
+    @Override
+    public void close() throws IOException {
+        ncd.close();
+    }
+
+    @Override
+    public long getLastModified() {
+        return 0;
+    }
+
+    @Override
+    public void setFileCache(FileCacheIF fileCache) {
+        ncd.setFileCache(fileCache);
+    }
+
+    @Override
+    public void getDetailInfo(Formatter sf) {
+        ncd.getDetailInfo(sf);
+    }
+
+    /**
+     * the name of the dataset is the last part of the location
+     * @return the name of the dataset
+     */
+    public String getName() {
+        String loc = ncd.getLocation();
+        int pos = loc.lastIndexOf('/');
+        if (pos < 0)
+            pos = loc.lastIndexOf('\\');
+        return (pos < 0) ? loc : loc.substring(pos+1);
+    }
+
+    /**
+     * @return the underlying NetcdfDataset
+     */
+    public NetcdfDataset getNetcdfDataset() {
+        return ncd;
+    }
+
+    /**
+     * This is a set of GeoGrids with the same GeoCoordSys.
+     */
+    public static class SimpleGeometryCovSet  {
+
+        private SimpleGeometryCS gcc;
+        private List<SimpleGeometryCoverage> covs = new ArrayList<>();
+
+        private SimpleGeometryCovSet(SimpleGeometryCS gcc) {
+            this.gcc = gcc;
+        }
+        
+        private void add(SimpleGeometryCoverage cov) {
+            covs.add(cov);
+        }
+
+        /**
+         * Get list of GeoGrid objects
+         */
+        public List<SimpleGeometryCoverage> getGrids() {
+            return covs;
+        }
+
+        /**
+         * all GridDatatype point to this GridCoordSystem
+         */
+        public SimpleGeometryCS getSGCoordSystem() {
+            return gcc;
+        }
+

```

Comment:
Does anything rely on this method? If not, could it be removed?

Reply:
removed.